### PR TITLE
[9.0][MIG] Import Postfinance Bank Statements

### PR DESCRIPTION
**Pull request by ecino**
This takes the Postfinance Bank Statement Parser that was in the module `l10n_ch_account_statement_base_import` module in version 8 and adapt it for version 9.

- [x] Depends on https://github.com/OCA/bank-statement-import/pull/110

### CODE REVIEW

File: l10n_ch_bank_statement_import_postfinance/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (yvaucher) [inline l10n_ch_bank_statement_import_postfinance/__init__.py]: You can remove headers from `__init__.py` files

File: l10n_ch_bank_statement_import_postfinance/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (yvaucher) [inline l10n_ch_bank_statement_import_postfinance/__openerp__.py]: Please use the short header version and update the copyright year to 2014-2017

See https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L1-L3

File: l10n_ch_bank_statement_import_postfinance/static/src/js/account_statement_reconciliation.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/* This is Javascript extension of module account
```
Comment (yvaucher) [inline l10n_ch_bank_statement_import_postfinance/static/src/js/account_statement_reconciliation.js]: Add license in this header

https://github.com/OCA/maintainer-tools/blob/master/template/module/static/src/js/web_module_name.js#L1-L2

**Review by yvaucher — CHANGES_REQUESTED** [review]
Make sure to get the whole history:

https://github.com/OCA/l10n-switzerland/commits/8.0/l10n_ch_account_statement_base_import

**Comment by ecino** [comment]
@yvaucher Not sure how to include the history of v8. Isn't it already in the commits history of 9.0 branch? Any hint is welcome

**Comment by yvaucher** [comment]
@ecino Here you will find a little how to: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0#howto

The link is available in https://github.com/OCA/l10n-switzerland/issues/172

**Comment by ecino** [comment]
@yvaucher Thanks. I hope it's what you requested... a lot of commits are now in this PR.

**Comment by yvaucher** [comment]
@ecino Yes perfect for the history.

**Comment by ecino** [comment]
Need first [camt parser PR to be merged](https://github.com/OCA/bank-statement-import/issues/110), in order to adapt this module so that Parser inheritance is not conflicting with other modules that may as well change the camt parser.

**Comment by yvaucher** [comment]
@ecino I added the dependency in the description of this PR

**Comment by yvaucher** [comment]
OCA/bank-statement-import#110 has been merged

**Review by yvaucher — APPROVED** [review]

**Comment by yvaucher** [comment]
@ecino could you amend your last commit to make a rebuild on travis ?

**Comment by ecino** [comment]
I made changes based on OCA 110. Tests are passing.

**Comment by ecino** [comment]
@yvaucher can we merge now?

**Comment by ecino** [comment]
@yvaucher Still waiting to merge

**Pull request merged**